modules/distribution/src/repository/resources/conf/templates/repository/conf/identity/charon-config.xml.j2

```diff
@@ -18,13 +18,21 @@
   -->
 
 <provisioning-config>
-    {% for key,value in identity.charon.properties.items()%}
-    <Property name="{{key}}">{{value}}</Property>
-    {% endfor %}
+    <Property name="user-schema-extension-enabled">{{charon.properties.user_schema_extension.enabled}}</Property>
+    <Property name="bulk-maxOperations">{{charon.properties.bulk_operations.max}}</Property>
+    <Property name="bulk-maxPayloadSize">{{charon.properties.bulk_payload_size.max}}</Property>
+    <Property name="patch-supported">true</Property>
+    <Property name="documentationUri">http://example.com/help/scim.html</Property>
```

Comment:
Let's make this configurable and have wso2 doc link as default

Reply:
Fixed.